dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatMessage.cs

```diff
@@ -17,20 +17,27 @@ public abstract class ChatMessageBase
     /// <summary>
     /// Content of the message
     /// </summary>
-    public string Content { get; set; }
+    public string Content { get; set; } = string.Empty;
 
     /// <summary>
     /// Dictionary for any additional message properties
     /// </summary>
     public IDictionary<string, string>? AdditionalProperties { get; set; }
 
     /// <summary>
-    /// Creates a new instance of the <see cref="ChatMessageBase"/> class
+    /// Creates a new instance of the <see cref="ChatMessage"/> class
+    /// </summary>
+    public ChatMessage()
```

Comment:
Isn't this enough to get the Issue fixed? 

Why `ChatMessageBase` can't be abstract?

Reply:
The problem is that `ChatHistory` was a list of `ChatMessageBase` so deserialization fail. Once I make `ChatMessageBase` becomes non abstract it's the same as `ChatMessage`.

Reply:
We can resolve the Deserialization problem but indeed won't be able to serialize it back. 

Make sense